src/DefaultRenderer.js

```diff
@@ -135,21 +135,21 @@ export default class DefaultRenderer extends Component {
     }
   }
 
-  getPanHandlers(direction, props) {
-    return direction === 'vertical' ?
+  getPanHandlers = (direction, props) => (
```

Comment:
Unfortunately, the proposal for public class fields & static properties is only at Stage 2.

It should be a static function with the keyword `static`, not an arrow function.

Same thing on other methods.